atramhasis/alembic/versions/1a4d62b02630_199_sources.py

```diff
@@ -25,15 +25,15 @@ def upgrade():
     op.create_table('conceptscheme_source',
         sa.Column('conceptscheme_id', sa.Integer(), nullable=False),
         sa.Column('source_id', sa.Integer(), nullable=False),
-        sa.ForeignKeyConstraint(['conceptscheme_id'], ['conceptscheme.id'], ),
+        sa.ForeignKeyConstraint(['conceptscheme_id'], ['conceptscheme.id'], ondelete="cascade"),
```

Comment:
mag je alembic files aanpassen? moet dit dus niet in een apart upgrade bestandje?